### PR TITLE
Fixed Bug #585 and enhances the behaviour to use reachable yb servers

### PR DESCRIPTION
**Pull request by sanyamsinghal**
First commit: fixes bug #585
Second commit: change the import data behavior to use the reachable servers instead of using all and failing, if not all.

Testing:
1. Tested a scenario where the output of the yb_servers() gives the DNS of servers which is not resolvable from the client machine, so the provided `target-db-host` is just used for importData
2. Cluster on YB Cloud where LB is present
3. Normal cluster without a load balancer

### CODE REVIEW

**Review by rahulb-yb — APPROVED** [review]
lgtm

File: yb-voyager/src/utils/utils.go [hunk 1]
```diff
@@ -290,7 +290,8 @@ func LookupIP(name string) []string {
 
 	ips, err := net.LookupIP(name)
 	if err != nil {
-		ErrExit("Error Resolving name=%s: %v", name, err)
+		log.Infof("Error Resolving name=%s: %v", name, err)
+		return []string{name}
```
Comment (amit-yb) [inline yb-voyager/src/utils/utils.go:294]: This is sneaky! Return an empty slice if the name can't be resolved. But do not play with the semantics of a function. In this case, the callers (or at least someone who is reading the code) would expect that the `LookupIP()` is going to return a list of IP addresses. Please, fix this before merging.

Comment (sanyamsinghal) [reply yb-voyager/src/utils/utils.go:294]: ok, will fix it before merging

Comment (sanyamsinghal) [reply yb-voyager/src/utils/utils.go]: done

**Pull request merged**